app/core/validation/util.py

```diff
@@ -66,6 +68,22 @@ def write_documents_to_s3(
     )
 
 
+class ResultType(str, enum.Enum):
```

Comment:
Question asked in slack: https://climate-policy-radar.slack.com/archives/C02ET708QDP/p1707841004892509

Reply:
See thread, this cannot be remove now